src/index.js

```diff
@@ -48,6 +50,7 @@ class Dropzone extends React.Component {
       document.addEventListener('dragover', Dropzone.onDocumentDragOver, false)
       document.addEventListener('drop', this.onDocumentDrop, false)
     }
+    this.fileInputEl.addEventListener('click', this.onInputElementClick, false)
```

Comment:
I'm wondering why. It should not as far as I understand. I thought that's a React bug but I could not reproduce it in isolation. Ideas?